app/assets/stylesheets/govuk_publishing_components/components/govspeak/_button.scss

```diff
@@ -3,13 +3,10 @@
 // scss-lint:disable PlaceholderInExtend
 // sass-lint:disable placeholder-in-extend
 .gem-c-govspeak {
-  .button,
-  .govuk-button,
   .gem-c-button {
     @extend .govuk-button;
   }
 
-  .button-start,
   .govuk-button--start {
```

Comment:
Does this `.govuk-button--start` extend itself?

Reply:
Yes, but necessarily. The reason for this whole bit of Sass is to override the styles within govspeak elements for links. It's a bit inefficient, in that `.govuk-button--start` is already defined, and we're duplicating the CSS by doing this, but without it the styles for that class get overwritten by govspeak - the button has link text colour instead of white, etc. It seems illogical but it's a necessary evil.

Now that you mention this I wonder if there's a world where we don't need this - by being more thoughtful about styling links within govspeak. We should be able to detect if a link is a button and if not, apply the `govuk-link` class to it.

Reply:
`:not()` for the win!

Reply:
Yeah, something like that. But outside of the scope of this PR.